about-impl-days.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: "What is `impl Days`"
+permalink: /about_impl_days/
+---
+
+<div class="backdrop" style="background: url(/assets/rome/the-city.jpg) 50% 40%; padding: 0; margin-bottom: 1em;">
+  <div class="popout">
+    <section>
+      <h1>impl Days</h1>
+    </section>
+  </div>
+</div>
+
+{% comment %}
+<section markdown="1">
+## Todays Info
+{% include_relative impl_days_room_info.html %}
+</section>
+{% endcomment %}
+
+<section markdown="1">
+## about:impl_days
+
+{% comment %}
+TODO: add sponsors
+{% endcomment %}
+
+`impl Days` is the sibling event of RustFest&lt;Rome&gt;. `impl Days` is a full-day event for Rust contributors to
+collaborate in an active programming session. It will take place on the two days following the RustFest talks and
+workshops, on {{site.dates.impl_days_no_br}} {{site.dates.year}}.
+
+## History
+
+Born as a hacking and development experiment in the early RustFest editions, the "impl Days" has established as a
+tradition following each RustFest.
+
+The name refers to the first edition that took place during the 2017 "impl Period", therefore the name "impl Days" was a
+natural choice.  <a href="https://internals.rust-lang.org/t/the-impl-period-newsletter-2/6034"> Over those two days</a>,
+30 to 40 people came together to discuss, design, and implement changes to Rust itself as well its broader ecosystem.
+
+Plus, <a href="https://botbot.me/mozilla/rust-internals/2017-10-02/?msg=91818378&page=2">they got to build a Lego Death
+Star</a>.
+
+The Zurich `impl Days` was such a success that we decided to repeat the event in Paris and it was an even <a
+href="https://twitter.com/MozillaParis/status/1001434063264272384" target="_new">bigger success</a> (also, <a
```

Comment:
should this be `target="_blank"`?

Reply:
never really ever noticed any difference since +10 years :-)

But I was curious to investigate a little (on s.o.) and the consensus seems that anything is ok but `_blank` is technically more appropriate, so - yeah - since I'm going to commit other stuff, I'll amend this too :+1: